### PR TITLE
Wrap long post descriptions

### PR DESCRIPTION
**Pull request by tildehacker**
Hi,

I made this patch 3 months ago to wrap long post descriptions. But it's just a quick fix, I disabled the animation because it wasn't working as expected.

I'm submitting this pull request now as a workaround for issue #43.

Example:
wrapped-post-description

Best regards,

### CODE REVIEW

**Comment by bitbrain** [comment]
Hey @tildehacker, I am afraid I cannot merge this because the animation was the very reason I started this theme. 🤣

Could you try to make it working with animations?

**Comment by tildehacker** [comment]
Hi,

I used a JavaScript code snippet from https://www.w3schools.com/howto/howto_js_typewriter.asp. When JavaScript is disabled, a \<noscript\> block is used to render the post description without animation.

This is a GIF preview:
post-description-wrap

For a live preview, this branch is already merged on my [blog](https://tildehacker.com).

Please let me know if you think I need to make any further changes to this PR.

Kind regards,

**Comment by bitbrain** [comment]
@tildehacker awesome, thanks! Let do https://github.com/bitbrain/jekyll-dash/issues/44 first to fix these nasty build errors and then I'll approve and merge.

Great job, as always! 🔥

File: _layouts/post.html [hunk 1]
```diff
@@ -44,3 +47,20 @@ <h3>
 </div>
 {% endif %}
 {% include tagcloud.html %}
+
+<script>
+  let i = 0;
+  const text = '{{ page.description }}';
+  const speed = 50;
```
Comment (bitbrain) [inline _layouts/post.html:54]: @tildehacker can we somehow make this configurable via the `_config.yml`?

Comment (tildehacker) [reply _layouts/post.html]: Yes, I added a new variable called `animation_speed` in `_config.yml` to make it configurable.

**Pull request merged**